pkg/storage/stores/local/shipper.go

```diff
@@ -235,6 +243,7 @@ func (s *Shipper) syncLocalWithStorage(ctx context.Context) error {
 
 	for period := range s.downloadedPeriods {
 		if err := s.syncFilesForPeriod(ctx, period, s.downloadedPeriods[period]); err != nil {
+			s.metrics.filesDownloadFailures.Inc()
```

Comment:
Why don't you track success and failure via the same counter ? I think it gives better possibilities.